core/src/primitives/rectangle/mod.rs

```diff
@@ -246,6 +246,64 @@ impl Rectangle {
         Rectangle::zero()
     }
 
+    /// Returns the minimum sized `Rectangle` that envelopes both `self` and `other`.
+    ///
+    /// # Examples
+    ///
+    /// ## Envelope
+    ///
```

Comment:
The heading isn't needed, because there is only one example.
```suggestion
    /// ## Envelope
    ///
```